### DISCUSSION TITLE
1.19.0 – Support of Google motion photos

### DISCUSSION THREAD

**Discussion by Radiokot** [Radiokot/photoprism-android-client#89] [General]
<img align="right" width=130 src="https://github.com/Radiokot/photoprism-android-client/assets/5675681/48c25c9c-ba3d-4c32-86c8-e83455b91b3c"/>

The recent [PhotoPrism September update](https://docs.photoprism.app/getting-started/updates/) brings indexing of Samsung and Google live (motion) photos. With version 1.19.0 the gallery gets support of Google photos, therefore supporting both formats now. If you already have live photos in your gallery, after updating the server, make sure to launch library re-scan. You can also test this feature on the [public demo](https://demo.photoprism.app/library/browse).
For me, Samsung shots with the still image taken at the end of the video look much more appealing than the Google ones, where the video looks to be recorder after the still image is captured. What do you think?

**Comment by dfoxg** [comment]
Thanks for adding this one!!

What do you think about adding a feature to rewatch the live video? Maybe a long press on the image, after it was opened? I think that's the behaviour on a iPhone.

One small thing I also noticed right at the moment: the selection view doesn't end when no more picture is selected. Maybe the app should close the selection with when the selection count is equals 0.

**Reply by Radiokot** [reply]
It totally makes sense to get out of selection mode if you manually deselect all items. I've just never done that. Will add this. The ability to view the full video of a live photo is also nice to have, I like the long press idea. Thanks a lot, Daniel!
